Dockerfile

```diff
@@ -1,9 +1,7 @@
-ARG PHP_VERSION=7.4
-ARG PHP_TARGET=php:${PHP_VERSION}-cli
+# add amd64 platform to support Mac M1
+FROM --platform=linux/amd64 shivammathur/node:latest-amd64
```

Comment:
If we're using `latest`, does this even respect `PHP_VERSION`?

Reply:
> If we're using latest, does this even respect PHP_VERSION?

I don't think we can pass the PHP_VERSION to `shivammathur/node`. It installs PHP from `5.6` to `8.1` automatically. But we need `PHP_VERSION` to enable extensions.

Reply:
> It installs PHP from 5.6 to 8.1 automatically

So it installs **all** PHP versions? That seems weird. I thought there are version-specific images.

Reply:
> So it installs all PHP versions?

I even checked the container source code, [You can see it is installing all of the PHP versions. ](https://github.com/shivammathur/node-docker/blob/main/Dockerfile_bionic#L88)

Reply:
I see. Aren't there version-specific images?

Reply:
> Aren't there version-specific images?

I couldn't find any.

Reply:
Okay, that's fine then. We can use `node` with this approach

Reply:
Resolving the review then.